### ISSUE TITLE
False positive: eslint-import-resolver-typescript

### ISSUE THREAD

**Issue by slambdev** [depcheck/depcheck#603]
### Bug Description

We're getting some false positives with the `eslint-import-resolver-typescript` module in our Typescript project.

### Code snippets

How we're using depcheck
```
npx depcheck . --ignore-bin-package=true --specials=eslint,tslint,jest,ttypescript
```

Where the dependency is used:

It's used in our eslintrc.js file as described here: 
https://www.npmjs.com/package/eslint-import-resolver-typescript

Where the dependency is listed in package.json:

```
    "devDependencies": {
       ....
        "@typescript-eslint/eslint-plugin": "^1.11.0",
        "@typescript-eslint/parser": "^1.13.0",
        "eslint": "^5.3.0",
        "eslint-config-airbnb-base": "13.2.0",
        "eslint-config-prettier": "^6.0.0",
        "eslint-import-resolver-typescript": "^1.1.1",
        "eslint-plugin-import": "^2.22.1",
        "eslint-plugin-jest": "^22.7.2",
        "eslint-plugin-prettier": "^3.1.1",
        "eslint-plugin-simple-import-sort": "^4.0.0",
        "jest": "^24.8.0",
        "nodemon": "^2.0.4",
        "prettier": "^1.18.2",
        "ts-jest": "^26.1.0"
    }
```

### Versions

- `node -v`: v14.8.0
- `npm -v`: 6.14.7
- `depcheck --version`: 1.2.0

**Comment by rumpl** [comment]
@slambdev thanks for reporting this, could you retry with the latest version, 1.3.1 please? Also a ‘depcheck —json’ would be great

**Comment by rumpl** [comment]
Thing is, I really think the resolver thing is fixed with the latest version

**Comment by slambdev** [comment]
Awesome. It works on the latest version! Thank you!

**Comment by martinek-stepan** [comment]
Same

> I still get a false positive for `eslint-import-resolver-typescript` as being missed. But it comes with a shared eslint config, where it's installed properly as a dependency.
> 
> > depcheck --specials=eslint
> 
> Missing dependencies
> * eslint-import-resolver-typescript: ./package.json
> 
> $ npm ls depcheck
> 
> └── depcheck@1.4.3
